Alcatraz/Installers/ATZPluginInstaller.m

```diff
@@ -71,6 +75,46 @@ - (NSString *)pathForInstalledPackage:(ATZPackage *)package {
                                        stringByAppendingString:package.extension];
 }
 
+- (BOOL)isPackageBlacklisted:(ATZPackage *)package {
+    if (![self isPackageInstalled:package]) {
+        return NO;
+    }
+
+    NSString *pluginManagerKey = [self xcodeDefaultsNonApplePluginsKey];
```

Comment:
If we switch to Alcatraz.app, this won't work anymore.
Do you think we could come up with a solution that reads the file externally?